engine/runtime-instrument-common/src/main/java/org/enso/interpreter/instrument/command/InvalidateModulesIndexCommand.java

```diff
@@ -33,6 +32,7 @@ public Future<BoxedUnit> executeAsynchronously(RuntimeContext ctx, ExecutionCont
           TruffleLogger logger = ctx.executionService().getLogger();
           long writeCompilationLockTimestamp = ctx.locking().acquireWriteCompilationLock();
           try {
+            ctx.jobControlPlane().stopBackgroundJobs();
```

Comment:
I find this confusing - we stop **all** background jobs only to abort **some** of them?

Reply:
It may look like an overkill. But here we stop running **new** background jobs for a brief moment to ensure that the deserialization jobs will not be executed before we clean the suggestions database. After the database is cleaned, we re-start the background jobs processing.